docs/hardware/lumig.md

```diff
@@ -159,11 +159,11 @@ The way the wavefronts are executed by a compute unit is the following:
   executed concurrently, the throughput remains constant at one instruction per
   wavefront per compute unit
 
-The compute units have 512 64-wide 4 bytes Vector General Purpose Registers
-(VGPRs).  In addition to these registers, the unit also provides access to low
-latency storage through a 64 kB Local Data Share (LDS). This shared memory is
-akin to NVIDIA's "shared memory" and is accessible to all threads within a block
-(workgroup). The programmer manages the LDS allocation. Additionally,
+The SIMD units have 512 64-wide 4 bytes Vector General Purpose Registers
```

Comment:
Some abbreviations are introduced, and some are not. Just checking, we assume users are familiar with SIMD, and this does not require an explanation? We only introduce the AMD specific abbreviations and concepts?

Reply:
I would say so.